documentation/Register-PnPManagementShellAccess.md

```diff
@@ -34,7 +34,19 @@ This cmdlet grants access to the tenant for the PnP Management Shell Multi-Tenan
 Register-PnPManagementShellAccess
 ```
 
-Launches the consent flow to grant the PnP Management Shell Azure AD Application delegate access to the tenant.
+### EXAMPLE 2
+```powershell
+Register-PnPManagementShellAccess -ShowConsentUrl
+```
+
+Launches the consent flow to grant the PnP Management Shell Azure AD Application delegate access to the tenant and also displays the consent URL which can be shared with Azure AD administrators, Global administrators or folks with necessary permissions.
+
+### EXAMPLE 3
+```powershell
+Register-PnPManagementShellAccess -ShowConsentUrl -TenantName yourtenant.onmicrosoft.com
+```
+
+Displays the consent URL which can be shared with Azure AD admininstrators, Global administrators or folks with necessary permissions.
```

Comment:
Maybe 'folks' is not the right word to use here...  also a typo: 'admininstrators'